### DISCUSSION TITLE
Welcome to Drop Discussions!

### DISCUSSION THREAD

**Discussion by EnZon3** [EnZon3/Drop#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by plesnerd** [comment]
Do you have an open beta or planned release date?

**Reply by EnZon3** [reply]
Hey-o, great question..

I don't have a set release date yet, especially this early in development but when I get more of the product finished I'll get a deadline set for then. At that point a release date would be announced.

And about the open beta, it'll be opened when the core functionality of Drop is finished.

**Comment by wietyy** [comment]
Super pumped you're making this. I'll be on the community as soon as it comes out 😄

**Reply by EnZon3** [reply]
Awesome! Thanks for all the enthusiasm!!
